qcelemental/tests/test_units.py

```diff
@@ -73,23 +118,77 @@ def test_unit_conversion_nist(from_unit, to_unit, expected):
         ("au_permittivity", "au_charge**2 / (au_length * au_energy)", "1.0"),
         ("au_time", "au_action / au_energy", "1.0"),
         ("au_velocity", "au_length * au_energy / au_action", "1.0"),
-        ("au_pressure", "J / m^3", "2.94210e13"),  # note precision loss due to absence in CODATA
+        ("au_pressure", "J / m^3", "2.94210e13"),
     ],
 )
-def test_unit_conversion_other(from_unit, to_unit, expected):
+def test_unit_conversion_other_2014(from_unit, to_unit, expected):
+    # Build values and quantize to the tolerance
+    constants = qcelemental.PhysicalConstantsContext("CODATA2014")
+    expected = Decimal(expected)
+    from_to_value = Decimal(constants.conversion_factor(from_unit, to_unit)).quantize(expected)
```

Comment:
We may want to abstract these away in testing functions.

Reply:
here's another round of test refactoring. no kidding that testing these cleanly is awkward. let me know if you've further simplifying suggestions. as this is the first multi-context example, and the simpler this gets, the easier future additions will be.

and definitely SQUASH merge